pkg/pods/hoist_launchable.go

```diff
@@ -130,19 +130,13 @@ func (hoistLaunchable *HoistLaunchable) Stop(serviceBuilder *runit.ServiceBuilde
 // All services will attempt to be started.
 func (hoistLaunchable *HoistLaunchable) Start(serviceBuilder *runit.ServiceBuilder, sv *runit.SV) error {
 
-	// if the service is new, building the runit services also starts them, making the sv start superfluous but harmless
-	err := hoistLaunchable.BuildRunitServices(serviceBuilder)
-	if err != nil {
-		return err
-	}
-
 	executables, err := hoistLaunchable.Executables(serviceBuilder)
 	if err != nil {
 		return err
 	}
 
 	for _, executable := range executables {
-		_, err := sv.Restart(&executable.Service)
+		_, err := sv.Start(&executable.Service)
```

Comment:
hmm okay. So if we remove a launchable from a pod we never explicitly stop it, it gets stop by rewriting the servicebuilder file and running servicebuilder.

On a slightly unrelated note, this seems weird for the preparer because even calling restart we're basically halting execution of any code after this line